{"uuid": "dfced11d-9470-4c2c-8543-6c278d14932b", "children": ["5e36ddca-6bb5-40ee-a555-edefb0fd523c"], "befores": [{"name": "android_mobile_management", "status": "broken", "statusDetails": {"message": "TypeError: argument should be a str or an os.PathLike object where __fspath__ returns a str, not 'NoneType'\n", "trace": "  File \"/Users/stanislavdudnik/Desktop/Git/qa_guru_22/venv/lib/python3.12/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/Users/stanislavdudnik/Desktop/Git/qa_guru_22/venv/lib/python3.12/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/Users/stanislavdudnik/Desktop/Git/qa_guru_22/venv/lib/python3.12/site-packages/pluggy/_callers.py\", line 102, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/stanislavdudnik/Desktop/Git/qa_guru_22/venv/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1149, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/stanislavdudnik/Desktop/Git/qa_guru_22/venv/lib/python3.12/site-packages/_pytest/fixtures.py\", line 916, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/Users/stanislavdudnik/Desktop/Git/qa_guru_22/tests/conftest.py\", line 38, in android_mobile_management\n    options = config.to_driver_options(context=context)\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/stanislavdudnik/Desktop/Git/qa_guru_22/config.py\", line 25, in to_driver_options\n    options.set_capability('app', file.abs_path_from_project(self.app))\n                                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/stanislavdudnik/Desktop/Git/qa_guru_22/utils/file.py\", line 7, in abs_path_from_project\n    .parent.parent.joinpath(relative_path)\n                   ^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/pathlib.py\", line 718, in joinpath\n    return self.with_segments(self, *pathsegments)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/pathlib.py\", line 385, in with_segments\n    return type(self)(*pathsegments)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/pathlib.py\", line 1163, in __init__\n    super().__init__(*args)\n  File \"/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/pathlib.py\", line 373, in __init__\n    raise TypeError(\n"}, "start": 1708332922249, "stop": 1708332922353}], "start": 1708332922249, "stop": 1708332922445}